src/hooks/useErrorToast/useErrorToast.tsx

```diff
@@ -29,7 +29,7 @@ export const useErrorHandler = () => {
           return [errorMsgTranslation, errorMsgTranslationOptions]
         }
 
-        return defaultErrorMsgTranslation
+        return [defaultErrorMsgTranslation]
```

Comment:
The actual fix. Everything else is `rename symbol`.

Reply:
Ah damn!